diagnostics/diagnostics.go

```diff
@@ -80,6 +80,19 @@ func RunDiagnostic(diagnostic structs.DiagnosticSpec, isCron bool, cronjob struc
 	akkeris.AddVar(newvar)
 	akkeris.UpdateVar(newvar)
 
+	// These environment variables tell the test suite that it is running in Akkeris CI
+	newvar.Setname = diagnostic.Job + "-" + diagnostic.JobSpace + "-cs"
+	newvar.Varname = "AKKERIS_CI"
```

Comment:
I think we should leave it as `AKKERIS_CI` because our product naming will be more like `Heroku CI` rather than `CircleCI` (with the space). Also our other environment variables have underscores in them